### PR TITLE
update version 0.13.2

### PR DESCRIPTION
**Pull request by dgarcia360**
* Added render support for mosaic restriction and metadata transactions
* Refactored transaction status command
* Updated version

Closes #121

### CODE REVIEW

File: src/service/transaction.service.ts [hunk 1]
```diff
@@ -188,37 +181,47 @@ export class TransactionService {
                         (modification.modificationType === AccountRestrictionModificationAction.Add ? 'Add' : 'Remove');
                     transactionFormatted += 'value:' + modification.value;
                 });
+        } else if (transaction instanceof MosaicGlobalRestrictionTransaction) {
+            transactionFormatted += 'MosaicGlobalRestrictionTransaction: ' +
+                'MosaicId:' + transaction.mosaicId.toHex() +
+                ' ReferenceMosaicId:' + transaction.referenceMosaicId.toHex() +
+                ' RestrictionKey:' + transaction.restrictionKey.toString() +
```
Comment (rg911) [inline src/service/transaction.service.ts]: this is going to be toHex() in v0.14.0 now. as restriction key has been changed to be in Hex format

File: src/service/transaction.service.ts [hunk 2]
```diff
@@ -188,37 +181,47 @@ export class TransactionService {
...
+                ' PreviousRestrictionValue:' + transaction.previousRestrictionValue.toString() +
+                ' PreviousRestrictionType:' + MosaicRestrictionType[transaction.previousRestrictionType] +
+                ' NewRestrictionValue:' + transaction.newRestrictionValue.toString() +
+                ' NewRestrictionType:' + MosaicRestrictionType[transaction.newRestrictionType];
+        } else if (transaction instanceof MosaicAddressRestrictionTransaction) {
+            transactionFormatted += 'MosaicAddressRestrictionTransaction: ' +
+                'MosaicId:' + transaction.mosaicId.toHex() +
+                ' RestrictionKey:' + transaction.restrictionKey.toString() +
```
Comment (rg911) [inline src/service/transaction.service.ts]: toHex()

File: src/service/transaction.service.ts [hunk 3]
```diff
@@ -188,37 +181,47 @@ export class TransactionService {
...
+                ' TargetAddress:' + transaction.targetAddress.pretty() +
+                ' PreviousRestrictionValue:' + transaction.previousRestrictionValue.toString() +
+                ' NewRestrictionValue:' + transaction.newRestrictionValue.toString();
+        } else if (transaction instanceof AccountMetadataTransaction) {
+            transactionFormatted += 'AccountMetadataTransaction: ' +
+                'TargetPublicKey:' + transaction.targetPublicKey +
+                ' ScopedMetadataKey:' + transaction.scopedMetadataKey.toString() +
```
Comment (rg911) [inline src/service/transaction.service.ts]: toHex()

File: src/service/transaction.service.ts [hunk 4]
```diff
@@ -188,37 +181,47 @@ export class TransactionService {
...
+                ' ValueSizeDelta:' + transaction.valueSizeDelta.toString() +
+                ' Value:' + transaction.value.toString();
+        } else if (transaction instanceof MosaicMetadataTransaction) {
+            transactionFormatted += 'MosaicMetadataTransaction: ' +
+                'TargetPublicKey:' + transaction.targetPublicKey +
+                ' ScopedMetadataKey:' + transaction.scopedMetadataKey.toString() +
```
Comment (rg911) [inline src/service/transaction.service.ts]: toHex()

File: src/service/transaction.service.ts [hunk 5]
```diff
@@ -188,37 +181,47 @@ export class TransactionService {
...
+                ' TargetMosaicId:' + transaction.targetMosaicId.toString() +
+                ' ValueSizeDelta:' + transaction.valueSizeDelta.toString() +
+                ' Value:' + transaction.value.toString();
+        } else if (transaction instanceof NamespaceMetadataTransaction) {
+            transactionFormatted += 'NamespaceMetadataTransaction: ' +
+                'TargetPublicKey:' + transaction.targetPublicKey +
+                ' ScopedMetadataKey:' + transaction.scopedMetadataKey.toString() +
```
Comment (rg911) [inline src/service/transaction.service.ts]: ToHex()

File: src/service/transaction.service.ts [hunk 6]
```diff
@@ -188,37 +181,47 @@ export class TransactionService {
...
+                ' TargetNamespaceId:' + transaction.targetNamespaceId.toString() +
+                ' ValueSizeDelta:' + transaction.valueSizeDelta.toString() +
+                ' Value:' + transaction.value.toString();
```
Comment (rg911) [inline src/service/transaction.service.ts]: already in string

File: src/service/transaction.service.ts [hunk 7]
```diff
@@ -188,37 +181,47 @@ export class TransactionService {
...
+            transactionFormatted += 'AccountMetadataTransaction: ' +
+                'TargetPublicKey:' + transaction.targetPublicKey +
+                ' ScopedMetadataKey:' + transaction.scopedMetadataKey.toString() +
+                ' ValueSizeDelta:' + transaction.valueSizeDelta.toString() +
+                ' Value:' + transaction.value.toString();
+        } else if (transaction instanceof MosaicMetadataTransaction) {
+            transactionFormatted += 'MosaicMetadataTransaction: ' +
+                'TargetPublicKey:' + transaction.targetPublicKey +
+                ' ScopedMetadataKey:' + transaction.scopedMetadataKey.toString() +
+                ' TargetMosaicId:' + transaction.targetMosaicId.toString() +
```
Comment (rg911) [inline src/service/transaction.service.ts]: toHex()

File: src/service/transaction.service.ts [hunk 8]
```diff
@@ -188,37 +181,47 @@ export class TransactionService {
...
+                ' ValueSizeDelta:' + transaction.valueSizeDelta.toString() +
+                ' Value:' + transaction.value.toString();
+        } else if (transaction instanceof NamespaceMetadataTransaction) {
+            transactionFormatted += 'NamespaceMetadataTransaction: ' +
+                'TargetPublicKey:' + transaction.targetPublicKey +
+                ' ScopedMetadataKey:' + transaction.scopedMetadataKey.toString() +
+                ' TargetNamespaceId:' + transaction.targetNamespaceId.toString() +
```
Comment (rg911) [inline src/service/transaction.service.ts]: toHex()

File: src/service/transaction.service.ts (hunk 6 above)
Comment (rg911) [inline src/service/transaction.service.ts]: already in string

File: src/service/transaction.service.ts (hunk 1 above)
Comment (dgarcia360) [reply src/service/transaction.service.ts]: Let's merge it as it is, and I'll take this into consideration when applying 0.14.

**Pull request merged**